packages-new/components/menu/src/components/osds-menu-item/osds-menu-item.tsx

```diff
@@ -0,0 +1,22 @@
+import { Component, Element, h, Host } from '@stencil/core';
+import { OdsMenuItemController } from './core/controller'
+
+import { HTMLStencilElement } from '@stencil/core/internal';
```

Comment:
use `import type { ...` for TS only code
(and move import type to the top of the imports)